src/main/java/levi9/VehicleService/service/impl/RentServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package levi9.VehicleService.service.impl;
+
+import levi9.VehicleService.model.Rent;
+import levi9.VehicleService.repository.RentRepository;
+import levi9.VehicleService.service.RentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class RentServiceImpl implements RentService {
+    @Autowired
+    private RentRepository rentRepository;
+
+    @Override
+    public boolean isVehicleRented(Long vehicleId){
+        List<Rent> allRents = rentRepository.findAll();
```

Comment:
No need to retrieve all vehicle but only one with that id. This can be performance issue when there are a lot more data